### PR TITLE
Add a env sample

### PR DESCRIPTION
**Pull request by gabrielgene**

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-csharp?pullRequest=52) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-csharp?pullRequest=52) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-csharp?pullRequest=52) it.</sub>

File: .env_sample [hunk 1]
```diff
@@ -0,0 +1 @@
+export SENDGRID_API_KEY=''
```
Comment (mbernier) [inline .env_sample]: @jericho, can you please help me out - you know csharp better than me. WAY better. What's the most appropriate way to create an environment file for C# on windows?

Comment (mbernier) [reply .env_sample]: @gabrielgene Can you set this file up like this one? https://github.com/sendgrid/sendgrid-csharp/pull/599/files#diff-04c6e90faac2675aa89e2176d2eec7d8R90

Comment (Jericho) [reply .env_sample]: @mbernier `env` files are a linux and/or docker concept aren't they? Anyway, I'm not familiar with them. The way I personally deal with environment variables is to have a `ps1` powershell file that I execute once to set my variable (this file contains lines such as: `Environment.SetEnvironmentVariable("the_name_of_my_variable", "the_value");`) and then I can retrieve the content of these variables in my C# code using `var value = Environment.GetEnvironmentVariable("the_name_of_my_variable");`

Comment (mbernier) [reply .env_sample]: NEAT, thank you @Jericho

**Comment by thinkingserious** [comment]
Hello @gabrielgene,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**